storage/innobase/xtrabackup/src/xbcloud/http.h

```diff
@@ -344,7 +344,7 @@ class Http_client {
       CURLcode::CURLE_GOT_NOTHING,       CURLcode::CURLE_OPERATION_TIMEDOUT,
       CURLcode::CURLE_RECV_ERROR,        CURLcode::CURLE_SEND_ERROR,
       CURLcode::CURLE_SEND_FAIL_REWIND,  CURLcode::CURLE_PARTIAL_FILE,
-      CURLcode::CURLE_SSL_CONNECT_ERROR, CURLcode::CURLE_HTTP2};
+      CURLcode::CURLE_SSL_CONNECT_ERROR, CURLcode::CURLE_OBSOLETE16};
```

Comment:
Can you please add a comment? It looks weird to add obselete16 as retriable error code :)